commands/docker-compose.ts

```diff
@@ -100,3 +100,8 @@ export function composeUp(dockerComposeFileUri?: vscode.Uri, selectedComposeFile
 export function composeDown(dockerComposeFileUri?: vscode.Uri, selectedComposeFileUris?: vscode.Uri[]) {
     compose('down', 'to take down', dockerComposeFileUri, selectedComposeFileUris);
 }
+
```

Comment:
Have you tried running this from the command palette?  I think you'll find something undesirable happen.

Reply:
Yes. I don't know of any other way to run docker compose Up or Down (the package.json only specifies them in the commands section, not in any context menu). 

The undesirable side-effect is that I'm asked to choose a compose file _once for each of the compose down and up operations_. 
@fiveisprime, is that a critical point of friction?